docs/ref-info/ref-operators-public-private.md

```diff
@@ -17,7 +17,14 @@ All instances of the Operator Service are designed with rigorous protections in
 
 There can be multiple instances of the Operator Service, operated by multiple participants. Each of these participants is known as a UID2 Operator.
 
-A UID2 Operator is simply an organization that runs the Operator Service. Operators receive and store encryption keys and salts from the UID2 Core Service, salt and hash <Link href="../ref-info/glossary-uid#gl-dii">DII</Link> to return raw UID2s, encrypt raw UID2s to generate UID2 tokens, and distribute UID2 token decryption keys.
+A UID2 Operator is simply an organization that runs the Operator Service. Operators periodically perform multiple tasks including the following:
```

Comment:
Nit: remove word "simply" - what's simple for us, may not be simple for the reader.